sauce/features/accounts/calendar-first-day/index.js

```diff
@@ -0,0 +1,67 @@
+import { Feature } from 'core/feature';
+import * as toolkitHelper from 'helpers/toolkit';
+
+export class CalendarFirstDay extends Feature {
+  constructor() {
+    super();
+    // Variables for tracking specific states
+    this.isCalendarOpen = false;
+    this.isReRendering = false;
+  }
+
+  shouldInvoke() {
+    return toolkitHelper.getCurrentRouteName().indexOf('account') !== -1;
```

Comment:
Since `invoke()` doesn't do anything, we can probably just `return false;` here, yeah?